src/components/vr_cooperation/include/vr_cooperation/commands/base_command_request.h

```diff
@@ -51,16 +51,24 @@ namespace commands {
  * @brief Base command class for requests
  */
 class BaseCommandRequest : public Command,
-  public event_engine::EventObserver<application_manager::MessagePtr, std::string> {
+  public event_engine::EventObserver<application_manager::MessagePtr, int> {
  public:
   /**
    * @brief BaseCommandRequest class constructor
    * @param message Message from mobile
-   **/
+   */
   explicit BaseCommandRequest(VRModule* parent,
                               const application_manager::MessagePtr& message);
 
   /**
+   * @brief BaseCommandRequest class constructor
+   * @param parent pointer to VRModule
+   * @param message Message from HMI
+   */
+  explicit BaseCommandRequest(VRModule* parent,
```

Comment:
doesn't make sens to use explicit with 2 params. It is only applicable for one param